tests/scripts/kubeadm.sh

```diff
@@ -130,8 +131,10 @@ case "${1:-}" in
             install_master
         ;;
         node)
-            if [ "$#" -eq 5 ]; then
-                install_node $3 $4 $5
+            if [[ $KUBE_VERSION == v1.8* ]] && [ "$#" -eq 7 ]; then
+                install_node $3 $4 $5 $6 $7
+            elif [ "$#" -eq 5 ]; then
+                install_node $3 $4 $5 "" ""
```

Comment:
if you sent $6 and $7 here, wouldn't they be empty anyway? could you just always pass this?
`install_node $3 $4 $5 $6 $7`

Reply:
done.